hail_scripts/reference_data/config.py

```diff
@@ -13,6 +25,26 @@ def predictor_parse(field: hl.StringExpression):
     return field.split(';').find(lambda p: p != '.')
 
 
+def clinvar_custom_select(ht):
+    selects = {}
+    clnsigs = parsed_clnsig(ht)
+    selects['pathogenicity'] = hl.if_else(
```

Comment:
The idea here was to split the string parsing and enum mapping apart so that the framework enum mapping could do its job here.  So we have the step here which first selects `pathogenicity` and `assertion` as a `hl.tstr` or `hl.tarray(hl.tstr)`, then the enum mapping step which will map to `pathogenicity_id` and `assertion_ids`.  

`conflictingPathogenicities` is a special snowflake that both parses and enum maps together and I left it as such with a comment.